src/scripts/shellcheck.sh

```diff
@@ -1,114 +1,15 @@
-set +e
-# Set colors for terminal output
-term_red=$'\e[1;31m'
-term_green=$'\e[1;32m'
-term_end=$'\e[0m'
-
-ORB_SOURCE_FILES=$(find "${SOURCE_DIR}" -type f -name "*.yml" -o -name "*.yaml")
-SCRIPT_FILES=$(find "${SOURCE_DIR}" -type f -name "*.sh")
-
-SC_ERROR_COUNT=0
+if ! command -v shellcheck &> /dev/null
+then
+    if [[ $EUID == 0 ]]; then export SUDO=""; else export SUDO="sudo"; fi
+    $SUDO apt-get update
+    $SUDO apt-get install shellcheck
```

Comment:
This assumes apt-based executor is used. See https://github.com/CircleCI-Public/shellcheck-orb/blob/2dot0/src/scripts/install.sh - At least Alpine should be accounted for I think (although I realize we'll be removing this soon anyways).